src/screens/HomeScreen/styles.ts

```diff
@@ -0,0 +1,62 @@
+import {StyleSheet} from 'react-native';
+
+export const HomeViewStyle = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: 8,
+  },
+  heading: {
+    textAlign: 'center',
+    color: '#0034a5',
+    fontSize: 35,
+    fontWeight: 'bold',
+    marginBottom: 30,
+  },
+  activeButton: {
+    userSelect: 'none',
+    paddingTop: 20,
+    paddingBottom: 20,
+    paddingRight: 40,
+    paddingLeft: 40,
+    textAlign: 'center',
+    borderWidth: 2,
+    borderRadius: 20,
+    marginTop: 20,
+    marginBottom: 30,
+    marginRight: 'auto',
```

Comment:
Can we add marginVertical instead of including both margin-left and right? The number of lines may reduce.

Reply:
yes we can , I ll fix this.